src/components/map/Map.tsx

```diff
@@ -365,7 +406,18 @@ const styles = StyleSheet.create({
     right: normalize(15),
     width: normalize(48),
     zIndex: 1
-  }
+  },
+  markerContainer: {
+    alignItems: 'center',
+    justifyContent: 'center',
+  },
+  activeMarkerContainer: {
+    // Define your styles for active marker here
+  },
+  map: {
+    width: '100%',
+    height: '100%',
+  },
```

Comment:
What does `map` style contribute to us?

Reply:
fills the available container so this styling could be removed from all other files where the map component is used, since the styling is reoccurring